stable/datadog/values.yaml

```diff
@@ -15,13 +15,15 @@ daemonset:
   ## Bind ports on the hostNetwork. Useful for CNI networking where hostPort might
   ## not be supported. The ports will need to be available on all hosts. Can be
   ## used for custom metrics instead of a service endpoint.
+  ## WARNING: Make sure that hosts using this are properly firewalled otherwise
+  ## otherwise metrics and traces will be accepted from any host able to connect to this host.
```

Comment:
oops, missed that. otherwise is on both lines

Reply:
good eye, my brain totally filtered that out